Leanplum-SDK/Classes/Features/Inbox/LPInbox.m

```diff
@@ -508,6 +509,9 @@ - (NSArray *)allMessages
         [messages addObject:[self messageForId:messageId]];
     }
     LP_END_TRY
+    
+    [[LPCountAggregator sharedAggregator] incrementCount:@"all_messages_inbox"];
```

Comment:
could you add LPCountAggregator as an instance variable / dependency to this class too?